src/pages/Home/HomePage.jsx

```diff
@@ -0,0 +1,105 @@
+import { Link } from "react-router-dom";
+import { Navbar, Footer } from "../../components/index";
+import "../../styles/homePage.css"
+
+export function Home(){
+    return(
+        <div>
+            <Navbar/>
+            <div className="cover-image">
+                <Link to="/products">
+                    <img className="img-responsive" src="https://res.cloudinary.com/dxsodik0q/image/upload/v1648042632/Sneaker%20Street/cover-image_xaanzu.jpg" alt="cover-image" />
+                </Link>
+            </div>
+            <div className="home-wrapper">
+                <div className="home-heading">
+                    <p>NEW ARRIVALS</p>
+                </div>
+                <div className="new-arrival">
+                    <a>
+                        <div className="grid-category">
+                            <img src="https://res.cloudinary.com/dxsodik0q/image/upload/v1648042630/Sneaker%20Street/new-arrival-1_fxnt2k.jpg" className="img-responsive" alt="" />
+                            <div className="product-detail">
+                                <div className="brand-name">Nike</div>
+                                <div className="product-name">NMD_R1 - CORE BLACK</div>
+                                <div className="price-tag">₹12,999</div>
+                            </div>
+                        </div>
+                    </a>
+                    <a>
```

Comment:
Better to divide Hero part and category sections into different components and then only added those components here. Will look much cleaner.

Reply:
Can try this. Thanks for suggestion